### PR TITLE
Fix specification link

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
$subject

## Examples
N/A

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

**Review by kaneeldias — APPROVED** [review]

File: docs/spec/spec.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ _Issue_: [#2289](https://github.com/ballerina-platform/ballerina-standard-librar
 
 # Introduction
 
-This is the specification for MySQL standard library, which the functionality required to access and manipulate data 
-stored in a MySQL database in the [Ballerina programming language](https://ballerina.io/), which is an open-source 
+This is the specification for the MySQL standard library, which the functionality required to access and manipulate data 
+stored in an MySQL database in the [Ballerina programming language](https://ballerina.io/), which is an open-source
```
Comment (BuddhiWathsala) [inline docs/spec/spec.md:12]:
```suggestion
stored in a MySQL database in the [Ballerina programming language](https://ballerina.io/), which is an open-source 
```

Comment (niveathika) [reply docs/spec/spec.md:12]: change similar to what praneesha proposed, https://github.com/ballerina-platform/module-ballerina-sql/pull/489#discussion_r787392548

Comment (BuddhiWathsala) [reply docs/spec/spec.md:12]: SQL has a debate - [see this blog](https://oracle-base.com/blog/2015/01/02/a-sql-or-an-sql/#:~:text=According%20to%20the%20Oracle%20docs,docs%20go%20the%20other%20route.). But according to the Oracle documentation it is `a SQL`.

However, in MySQL, there is no debate, we should use `a MySQL` - [see here](https://en.wikipedia.org/wiki/MySQL)

@praneesha what is your opinion?

Comment (praneesha) [reply docs/spec/spec.md:12]: Better to use `a MySQL` and `an SQL` for correct grammar.

File: docs/spec/spec.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ programming language for the cloud that makes it easier to use, combine, and cre
 
 # 1. Overview
 
-This specification elaborates on usage of MySQL `Client` interface to interface with an MySQL database.
+This specification elaborates on usage of the MySQL `Client` interface to interface with an MySQL database.
```
Comment (praneesha) [inline docs/spec/spec.md:26]:
```suggestion
This specification elaborates on the usage of the MySQL `Client` interface to interface with a MySQL database.
```

File: docs/spec/spec.md (hunk 1 above)
Comment (niveathika) [reply docs/spec/spec.md]: Ack

**Review by praneesha — APPROVED** [review]

**Review by BuddhiWathsala — APPROVED** [review]

**Review by daneshk — APPROVED** [review]

**Review by pubudu91 — APPROVED** [review]

**Pull request merged**